api/filtering/filtering.py

```diff
@@ -113,7 +113,8 @@ def _object_filter_builder(input_object, spec):
                     field_value,
                     child_filter,
                     operation,
-                    child_spec,
+                    is_array=None,
```

Comment:
Making a few suggestions for `is_array` to be False instead of None, since the naming and usage implies a boolean
```suggestion
                    is_array=False,
```